### PR TITLE
Add cathoderaytube adjustments

### PR DESCRIPTION
**Pull request by Szpadel**
#### Indexer/Tracker
cathoderaytube

#### Description
Replace S01 -> Season 1 as that convention is used on tracker

Changes untested as I wasn't able to find instructions how to rebuild with new version

### CODE REVIEW

**Comment by bakerboy448** [comment]
> Changes untested as I wasn't able to find instructions how to rebuild with new version

- [ ] Testing required.

There's nothing to build. indexer definitions are not built nor bundled anywhere.

use a custom def for testing
https://wiki.servarr.com/prowlarr/indexers#adding-a-custom-yml-definition

**Comment by Szpadel** [comment]
@bakerboy448 Thank you for link, I was able to test this and it looks like it works now - is able to find releases, as before none of the series could be found, there is still issue with sonarr not supporting multi season releases, but this is outside the scope. You can consider this PR as tested now.


Could this url be added to README.md? This isn't obvious for first time contributors and, at least for me, this wasn't easily discoverable

**Comment by bakerboy448** [comment]
Thanks!

added to readme as well - as part of the v7 readme update

**Review by bakerboy448 — APPROVED** [review]

File: definitions/v4/cathoderaytube.yml [hunk 1]
```diff
@@ -95,6 +100,13 @@ search:
           args: \[(\d+)\]
     title:
       selector: a[href^="/torrents.php?id="]
+      filters:
+        - name: re_replace # Seasons X-Y --> SX-SY
+          args: ["(?i)seasons?\\s*(\\d+)\\s*-\\s*(\\d+)", "S$1-S$2"]
+        - name: re_replace # Season X --> S0X
+          args: ["(?i)season\\s*(\\d+)", "S$"]
```
Comment (mynameisbogdan) [inline definitions/v4/cathoderaytube.yml:107]: I think there's a typo, here shouldn't the replacement be `S$1`?

**Pull request merged**